DevelopGitHub/GitHubExtension/GitHubExtension.Activity.Internal.WebApi/Commands/ContextActivityCommand.cs

```diff
@@ -0,0 +1,21 @@
+﻿using GitHubExtension.Activity.DAL;
+
+namespace GitHubExtension.Activity.Internal.WebApi.Commands
+{
+    public class ContextActivityCommand : IContextActivityCommand
```

Comment:
You have ActivityContext. Can you explain why you are named your command class ContextActivityCommand instead of ActivityContextCommand?